### PR TITLE
refactor(Makefile): Cleanup variables

### PR DESCRIPTION
**Pull request by provokateurin**
Trying to understand the Makefile with all these unnecessary variables is really hard, so I just removed them and moved almost everything inline.

### CODE REVIEW

**Review by joshtrichards — APPROVED** [review]
Made one small additional adjustment that was needed. Passes all tests now.

**Comment by provokateurin** [comment]
Maybe @bigcat88 you can review this as well?

**Comment by bigcat88** [comment]
> Made one small additional adjustment that was needed. Passes all tests now.

Thank you again


> Maybe @bigcat88 you can review this as well?

Looks good, let's merge this

**Review by bigcat88 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,19 +1,11 @@
-poetry=poetry
-poetry_run=$(poetry) run
-python=$(poetry_run) python
-pyresttest=$(poetry_run) pyresttest
-coverage=$(poetry_run) coverage
 manage-script=$(CURDIR)/manage.py
-manage=$(poetry_run) $(manage-script)
-db=sqlite
```
Comment (bigcat88) [inline Makefile:8]: please return "$db" variable

**Review by bigcat88 — CHANGES_REQUESTED** [review]

**Review by bigcat88 — APPROVED** [review]
now the tests for `pgsql` is run on `pgsql` and not on `sqlite`

**Pull request merged**